packages/jetpack-blocks/babel.config.js

```diff
@@ -0,0 +1,89 @@
+/** @format */
+const _ = require( 'lodash' );
+
+const targets = { browsers: [ 'last 2 versions', 'Safari >= 10', 'iOS >= 10', 'ie >= 11' ] };
+
+const config = {
+	presets: [
+		[
+			'@babel/env',
+			{
+				targets,
+				useBuiltIns: 'entry',
+				// allows es7 features like Promise.prototype.finally
+				shippedProposals: true,
+				// Exclude transforms that make all code slower, see https://github.com/facebook/create-react-app/pull/5278
+				exclude: [ 'transform-typeof-symbol' ],
+			},
+		],
+		'@babel/react',
+	],
+	plugins: _.compact( [
+		'@babel/plugin-proposal-class-properties',
+		'@babel/plugin-proposal-export-default-from',
+		'@babel/plugin-proposal-export-namespace-from',
+		'@babel/plugin-syntax-dynamic-import',
+		[
+			'@babel/transform-runtime',
+			{
+				corejs: false, // we polyfill so we don't need core-js
+				helpers: true,
+				regenerator: false,
+				useESModules: false,
+			},
+		],
+	] ),
+	overrides: [
+		{
+			test: [ './client/gutenberg/extensions', './packages/jetpack-blocks' ],
```

Comment:
These relative paths won't work here, but we do (kinda) need these plugins (long story -- it probably works right now because it's using some fallback JSX transform from the React preset instead, but we shouldn't rely on that).

This will be solved by #31464. For now, let's maybe just change this line to 

```suggestion
			test: [ '.src' ],
```